hedera-node/configuration/preprod/application.properties

```diff
@@ -1,3 +1,4 @@
 contracts.chainId=298
 contracts.systemContract.updateCustomFees.enabled=true
 entities.unlimitedAutoAssociationsEnabled=true
+contracts.systemContract.updateNFTsMetadata.enabled=true
```

Comment:
Should we also enable it for other environments, i.e. previewnet?

Reply:
I would not enable this until the metadata HIP has been implemented.

Reply:
I think I would wait until the metadata HIP has been implemented before turning this on